addiply/addiply.py

```diff
@@ -0,0 +1,8 @@
+import sys
+
+num_cases = int(sys.stdin.readline().rstrip())
+
+for _ in range(num_cases):
+    arr = sys.stdin.readline().rstrip().split(" ")
+
+    print((int(arr[0]) + int(arr[1])), (int(arr[0]) * int(arr[1])))
```

Comment:
This is a good one liner that get exactly what you need for the solution in codequest! 
For clean code though its always good to store value so they're reusable. for example if we store int(arr[0]) as a variable x then we don't need to cast it twice. So for example the below is a little bit easier to read and maintain.

x = int(arr[0])
y = int(arr[1])
print((x+y), (x*y)) 

Again though for codequest what you have is perfect